raiden-ts/src/config.ts

```diff
@@ -116,6 +119,9 @@ export function makeDefaultConfig(
     pfs: '', // empty string = auto mode
     matrixExcessRooms: 3,
     pfsSafetyMargin: 1.0,
+    pfsMaxPaths: 3,
+    pfsMaxFee: parseEther('0.05') as UInt<32>,
```

Comment:
Yes, it is. We do assume it for this token, because SVT/RDN has 18 decimals on all networks. One can change it if needed, it's just a constant default.